extensions/ql-vscode/test/unit-tests/command-lint.test.ts

```diff
@@ -65,6 +65,12 @@ describe("commands declared in package.json", () => {
     contribContextMenuCmds.add(command);
   });
 
+  menus["view/item/context"].forEach((commandDecl: CmdDecl) => {
+    const { command } = commandDecl;
+    paletteCmds.delete(command);
+    scopedCmds.add(command);
+  });
+
```

Comment:
As mentioned above, I think the previous change was correct (deleting this bit, but adding `command.match(/^codeQLQueries\./)` to the "allowed prefixes"). 

We currently add a new prefix for each new panel, so this would be consistent with the others 😺